Dnn.AdminExperience/ClientSide/Extensions.Web/package.json

```
@@ -6,7 +6,8 @@
     "build": "set NODE_ENV=production&&webpack --mode production",

```

I saw this was added in the Licensing project, do we want this change everywhere?
```suggestion
    "build": "set NODE_ENV=production && yarn lint && webpack --mode production",
```

---

No that was a mistake on my part, the webpack config has its own eslint plugin that runs eslint during the build. We would actually have to remove it from where it was done that way. It does not hurt but eslint runs twice this way. Good catch. I am working on another PR, will try to do that later unless you can do the opposite suggestion in the browser.